sdk/core/System.ClientModel/tests/client/ModelReaderWriter/ServiceModels/AvailabilitySetData.Serialization.cs

```diff
@@ -85,6 +85,21 @@ private void Serialize(Utf8JsonWriter writer, ModelReaderWriterOptions options)
                 JsonSerializer.Serialize(writer, ProximityPlacementGroup);
             }
             writer.WriteEndObject();
+            if (options.Format != "W" && _serializedAdditionalRawData != null)
```

Comment:
this model previously are not implemented with the additional raw data during serialization and deserialization. Here we added it.